### PR TITLE
[Task][66487] Handle UI for Get Started Screen (On-Boarding Screen)

### PR DESCRIPTION
**Pull request by Lomahs1401**
## Related Tickets
- [#Task 66487: Handle UI for Get Started Screen](https://edu-redmine.sun-asterisk.vn/issues/66487)
## WHAT
- Handle UI (Click Next Button, Skip Intro) for Get Started Screen (On-Boarding Screen)
## Evidence (Screenshot or Video)
image
image

https://github.com/awesome-academy/FoodRecipeApp/assets/77973385/4264abb0-d4fc-4876-b934-4a2e97bdd4c1

## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://sal.vn/WtYPAy |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://sal.vn/piPoQn |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/example/foodrecipeapp/screen/onboarding/OnBoardingActivity.kt [hunk 1]
```diff
@@ -48,4 +78,66 @@ class OnBoardingActivity : BaseViewBindingActivity<ActivityOnBoardingBinding>()
...
+        if (index == binding.introSlider.size + 1) {
+            binding.btnNext.text = getString(R.string.get_started)
+            binding.tvSkipIntro.visibility = View.INVISIBLE
+        } else {
+            binding.btnNext.text = getString(R.string.next)
+            binding.tvSkipIntro.visibility = View.VISIBLE
+        }
+    }
+
+    inner class IntroSlide(val slogan: String, val content: String, val backgroundImg: Int)
```
Comment (ghost) [inline app/src/main/java/com/example/foodrecipeapp/screen/onboarding/OnBoardingActivity.kt]: để lại data class đi e

File: app/src/main/res/anim/slide_in_right.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android">
+
+</set>
```
Comment (ghost) [inline app/src/main/res/anim/slide_in_right.xml]: chưa set animation à e

Comment (Lomahs1401) [reply app/src/main/res/anim/slide_in_right.xml]: Done

File: app/src/main/java/com/example/foodrecipeapp/screen/onboarding/OnBoardingActivity.kt (hunk 1 above)
Comment (Lomahs1401) [reply app/src/main/java/com/example/foodrecipeapp/screen/onboarding/OnBoardingActivity.kt]: Done

**Review by ghost — APPROVED** [review]

**Pull request merged**